go2_robot_sdk/go2_robot_sdk/go2_driver_node.py

```
@@ -197,13 +198,23 @@ def __init__(self):
                 qos_profile)
+            self.create_subscription(
+                WebRtcReq,
+                'webrtc_req',
+                lambda msg: self.webrtc_req_cb(msg, "0"),
+                qos_profile)
         else:
             for i in range(len(self.robot_ip_lst)):
                 self.create_subscription(
                     Twist,
                     f'robot{str(i)}/cmd_vel_out',
                     lambda msg: self.cmd_vel_cb(msg, str(i)),
```

Ruff (0.8.2)

Function definition does not bind loop variable `i`

(B023)